### ISSUE TITLE
Sitting animation

### ISSUE THREAD

**Issue by MCLV-pandorabox** [mt-mods/homedecor_modpack#82]
digging the chair while seated, you remain attached to an object and unable to move.
~~If you happen to die on a chair, you are visibly sitting to others while you don't see that yourself.~~

**Comment by wsor4035** [comment]
updated the second one as thats fixed now, unless you have evidence otherwise. regarding the first, think im just going to handle digging the chair, voxelmanip/worldedit thats on whoever is doing it

**Comment by MCLV-pandorabox** [comment]
![screenshot_20241107_042541](https://github.com/user-attachments/assets/d1f3a2ba-bf7d-4564-92ea-d49d5f8b6f4b)

**Comment by wsor4035** [comment]
> ![screenshot_20241107_042541](https://private-user-images.githubusercontent.com/107920484/383804422-d1f3a2ba-bf7d-4564-92ea-d49d5f8b6f4b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5NTE4NTMsIm5iZiI6MTczMDk1MTU1MywicGF0aCI6Ii8xMDc5MjA0ODQvMzgzODA0NDIyLWQxZjNhMmJhLWJmN2QtNDU2NC05MmVhLWQ0OWQ1ZjhiNmY0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwN1QwMzUyMzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZDhmZTVlYjk4NTAxMWI2ZDNhYTM0YmM4ZGIyNDM1Mjk2NjBlOGYwMzRmZTJmYzgzMWRjODNkMjJmN2U4MTdiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.h2cdGnybYEWlTeLLiVnoJTDZRpLP1KWbs4UvV7CtKlM)

irrelevant, https://github.com/pandorabox-io/pandorabox-mods/pull/3884

**Comment by MCLV-pandorabox** [comment]
don't understand your comment. d8ae7a3 is the latest .  isn't it?

**Comment by wsor4035** [comment]
yes, which the server in question your "testing" on hasnt pulled in yet. hence your "test" is irrelevant

**Comment by MCLV-pandorabox** [comment]
oh, I thought it had. this github stuff is hard to follow for me. sorry bout that

**Comment by MCLV-pandorabox** [comment]
works like a charm

**Comment by SwissalpS** [comment]
> this github stuff is hard to follow

if the changes are merged to the pandorabox-meta repo, then you can compare the merge date with last reboot of the server. Simple enough once you know it :D
